doc/source/dev/dev_quickstart.rst

```diff
@@ -108,3 +108,23 @@ Other workflows
 
 This is only one possible way to set up your development environment out of
 many. For more detailed instructions, see the :ref:`contributor-toc`.
+
+.. note::
+
+    If you are having trouble building SciPy from source or setting up your
+    local development environment, you can try to build SciPy with GitHub
+    Codespaces. It allows you to create the correct development environment
+    right in your browser, reducing the need to install local development
+    environments and deal with incompatible dependencies.
```

Comment:
I would also add that Gitpod is still possible, though won't have a custom pre-built development environment.